docs/writing-tests/visual-testing.md

```diff
@@ -40,6 +40,13 @@ npx chromatic --project-token <your-project-token>
  
 </div>
 
+<div class="aside">
+ 
+ GOTCHA: If you're project has less than 2 commits chromatic publishing will fail while `Retrieving git information` add another commit and try again.
+ 
+</div>
```

Comment:
@cabbiepete we have a bit of room for improvement here. We could move this below the snippet with the text " Build 1 published...." and probably go with:

```html
<div class="aside">
💡 Before running Chromatic's CLI ensure you have at least two commits added to the repository to prevent build failures, as Chromatic relies on a full Git history graph to establish the baselines. Read more about baselines in Chromatic's <a href="https://www.chromatic.com/docs/branching-and-baselines"> documentation</a>
</div>
```

Reply:
@jonniegodes thanks done.